src/components/card/card.module.css

```diff
@@ -1,70 +1,72 @@
 .cardBody {
     position: relative;
     display: flex;
-    justify-content: space-evenly;
+    justify-content: space-between;
     flex-direction: column;
     align-items:center;
     width: 180px;
     height: 290px;
+    padding: 5% 5% 5% 5%;
     background-color: rgba(255, 255, 255, 0.5);
     border: 1px solid white;
     border-radius: 2px;
     text-align: center;
     top: 0;
     left: 0;
     transition: transform 1s;
+    overflow: hidden;
 }
 .hoverEffect:hover {
     box-shadow: 0 0 3px 2px rgba(255, 255, 255, 0.6);
     cursor: pointer;
 }
 .disabled {
-    width: 180px;
-    height: 290px;
-    z-index: 9;
-    background-color: rgba(0, 0, 0, 0.5);
-    position: absolute;
+    filter: brightness(50%);
 }
 .imgContainerDiv {
-    width: 160px;
-    height: 160px;
+    width: 100%;
+    height: 60%;
     background-color: rgba(0, 0, 0, 0.6);
     border-top-left-radius: 100%266%;
     border-top-right-radius: 100%266%; 
     border-bottom-left-radius: 100%266%;
     border-bottom-right-radius: 100%266%;
 }
 .img {
-    width: 160px;
-    height: 160px;
-    object-fit: fill;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
     background-color: rgba(0, 0, 0, 0.6);
     border-top-left-radius: 100%266%;
     border-top-right-radius: 100%266%; 
     border-bottom-left-radius: 100%266%;
     border-bottom-right-radius: 100%266%;
 }
 .titleSpan {
-    height: 42px;
+    height: 12%;
     display: flex;
     justify-content: center;
     align-items: center;
+    line-height: 1em;
 }
 .textSpan {
-    height: 60px;
-    display: flex;
+    height: 15%;
+    overflow-y: scroll;
```

Comment:
This change should be noted. It is to allow for the entire description block to be read by means of a scroll bar if an overflow occurs.